src/Writer/Stream.php

```diff
@@ -10,9 +10,11 @@
 namespace Zend\Log\Writer;
 
 use Traversable;
+use Zend\Db\Adapter\Exception\ErrorException;
```

Comment:
This should be removed; it's never referenced by the code. (I can do that on merge.)